### PR TITLE
Probable fix for issue #26

### PR DESCRIPTION
**Pull request by pmonks**
Probable (untested) fix for issue #26.  Please note that I haven't deployed the resulting container to Dockerhub, so this fix should not be considered "tested".

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,5 +1,7 @@
 FROM clojure:openjdk-14-tools-deps
 
+RUN apt-get update && apt-get install -y git
```
Comment (liquidz) [inline Dockerfile:3]: Could you change as below?
c.f. https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#run

```suggestion
RUN apt-get update && apt-get install -y git && rm -rf /var/lib/apt/lists/*
```

**Review by liquidz — CHANGES_REQUESTED** [review]
@pmonks Ah, sorry for that.
I tested in my local environment, and works well.

Could you change the only point I suggested?

**Comment by pmonks** [comment]
No worries - I'm just sorry I didn't catch this myself!

**Review by liquidz — APPROVED** [review]

**Comment by liquidz** [comment]
Thank you so much!
I'll deploy to dockerhub.

**Pull request merged**